images/java/src/main/java/io/iron/lambda/Launcher.java

```diff
@@ -131,6 +131,15 @@ private void runMethod(Method lambdaMethod, String payload, Class cls, String ha
         if (!processed) {
             System.err.println(String.format("Handler %s with simple, POJO, or IO(input/output) types not found", handlerName));
         }
+
+        try {
+            String jsonInString = ClassTypeHelper.gson.toJson(result);
+            System.out.println(jsonInString);
+        }
+        catch (Exception e) {
+            System.out.println("{\"error\": \"" + e + "\"}");
```

Comment:
It is possible for the exception message to have `"` double quotes, in which case the resulting JSON will be invalid. What you want to do is replace `e` with `gson.toJson(e.toString())`?